.github/ISSUE_TEMPLATE/✨-feature-request.md

```diff
@@ -1,11 +1,3 @@
----
-name: "✨ Feature Request"
-about: A new feature or change to existing functionality
-title: "✨ Feature Request"
-labels: feature
-assignees: ''
----
```

Comment:
are all these deletions desired? I found auto-labelling with `feature` kind of neat

Reply:
Unintentional! Thanks.